config.yaml

```diff
@@ -22,14 +22,20 @@ logging:
 
 # Statsd metrics reporter
 metrics:
-  type: statsd
-  host: localhost
-  port: 8125
+  prometheus:
+    type: prometheus
+    port: 9000
+  statsd:
+    type: statsd
+    host: localhost
+    port: 8125
```

Comment:
hm.... changing the config spec is not really great... This means that whenever a service updates to a new service-runner version, it needs to update it's config, while it might not yet switch to prometheus for actual metrics collection. Can we still support the old config and maybe log a deprecation warning if it's detected?

Reply:
Great point.  It's probably best not to change the spec at all.